statsmodels/discrete/tests/test_constrained.py

```diff
@@ -531,29 +521,31 @@ def setup_class(cls):
 def junk():
     # Singular Matrix in mod1a.fit()
 
-    formula1 = 'deaths ~ smokes + C(agecat)'
-
-    formula2 = 'deaths ~ C(agecat) + C(smokes) : C(agecat)'  # same as Stata default
+    # same as Stata default
+    formula2 = 'deaths ~ C(agecat) + C(smokes) : C(agecat)'
 
-    mod = Poisson.from_formula(formula2, data=data, exposure=data['pyears'].values)
+    mod = Poisson.from_formula(formula2, data=data,
+                               exposure=data['pyears'].values)
 
-    res0 = mod.fit()
+    mod.fit()
 
-    constraints = 'C(smokes)[T.1]:C(agecat)[3] = C(smokes)[T.1]:C(agecat)[4]'
+    constraints = 'C(smokes)[T.1]:C(agecat)[3] = C(smokes)[T.1]:C(agec`at)[4]'
 
     import patsy
     lc = patsy.DesignInfo(mod.exog_names).linear_constraint(constraints)
     R, q = lc.coefs, lc.constants
 
-    resc = mod.fit_constrained(R,q, fit_kwds={'method':'bfgs'})
+    mod.fit_constrained(R, q, fit_kwds={'method': 'bfgs'})
```

Comment:
this is useless, calling something without catching return

same below

Reply:
smoke test

Reply:
If you want to avoid pep-8 unused variable, then it's better to just add an assert at the end, for example an isinstance check